developer-docs-site/docs/standards/object.md

```diff
@@ -0,0 +1,239 @@
+---
```

Comment:
It might be nice that every time something new is used in a code snippet, e.g. tje first time we use `Object<T>`, you include the import for `Object`.